src/smartpeak/source/core/Utilities.cpp

```diff
@@ -522,7 +522,7 @@ namespace SmartPeak
       }
 
       const std::string filename(entry.path().filename().string());
-      if (filename == "." || filename == "..") {
+      if (filename == "." || filename == ".." || (*(&filename.at(0))) == '.') {
```

Comment:
Was this the fix for a working executable on mac after calling cpack?

Reply:
Yes, to pass the test cases for `Utilities_test` but it didn't seem to have fixed the issue on macOS, I think we need a separate ticket to investigate why `Utilities::getPathnameContent` behaves differently on CircleCi instance.